src/components/ViewItem/ViewItem.scss

```
@@ -0,0 +1,35 @@
+@import '/src/styles/variables.scss';
+
+.viewitem {
+  display: flex;
+  justify-content: space-around;
+  margin: 40px 70px;
+
+  .clothesBox {
+    .defaultText {
+      &:nth-child(8) {
+        color: $font-color-black;
+        font-size: $font-size-lg;
+      }
+    }
+  }
```

```suggestion
.defaultText:nth-child(8) {
    color: $font-color-black;
    font-size: $font-size-lg;
  }
```

clothesBox와 defaultText 요소에 별다른 스타일이 없다면
위와 같이 수정할 수 있겠습니다.

---

수정 했습니다!